Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Twisting Nether

## Changelog:
- Characters with Demon Trait don't die, they return to the Twisting Nether and may be reborn later.
  - Added a background for the Twisting Nether.
  - Being in the Nether makes you isolated, you can't interact much with the world, can't be a target of schemes.
- Added the Incapable game concept.
- The Incapable and In Twisting Nether traits mention they make characters incapable.
- Satyrs look like man'ari for now.


## Developer changelog:
- Replace vanilla death effect with mod override death effect.
- Added `is_incapable_trigger` to replace vanilla trait check,

# Tasks:

<details>
<summary>Click to expand</summary>

- [x] Add Return to Nether effect
- [x] Create Return to Twisting Nether concept. Mentions when you can't return.
- [x] Add Killed Me opinion
- [x] Localize the event
- [x] Add stress, prestige level cost
- [x] Add event and character background
- [x] Relocalize
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/129456303-cb99663f-1428-4dea-ac92-4aabbd4bd2fa.png)

</details>

- [x] This event shouldn't happen
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/129459385-2e524aee-022a-4486-a933-aa0e1d26b9b2.png)

</details>

- [x] You shouldn't be able to plot against them
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/129485715-77095711-1758-4a85-b309-a8b377d531dc.png)

</details>

</details>


# How to test:
Start as a demon (like satyr) and try to die.
In the Debug mode, you can right-click your character and force someone to kill you.
Tell your opinion about price and balance.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/592/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here are the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
